### PR TITLE
Launcher: fix iomp5, add transparent hugepage and add debug option

### PR DESCRIPTION
**Pull request by ashokei**

### CODE REVIEW

**Comment by jgong5** [comment]
@liangan1 @EikanWang

**Comment by EikanWang** [comment]
@liangan1 May I know if there is any UT that can cover the launcher?

File: intel_pytorch_extension_py/launch.py [hunk 1]
```diff
@@ -212,22 +214,39 @@ def add_lib_preload(lib_type=None):
...
+           val = f.read().strip().split("[")[1].split("]")[0]
+           if val in ['always','never','madvise']:
+               return val
+    return None
+
+def set_thp(arg):
+    if not arg: return
+    if os.path.exists(THP_LOC):
+        os.system("sync;echo 3 > /proc/sys/vm/drop_caches")
+        os.system(f"echo {arg} > {THP_LOC}")
```
Comment (liangan1) [inline intel_pytorch_extension_py/launch.py:247]: is it general to all Linux platform about the setting for transparent hugepage ?

**Review by liangan1 — COMMENTED** [review]
There is no UT to cover the launcher.

**Comment by jgong5** [comment]
> There is no UT to cover the launcher.

@liangan1 We'd better add UT to cover the basic functionalities.

File: intel_pytorch_extension_py/launch.py (hunk 1 above)
Comment (ashokei) [reply intel_pytorch_extension_py/launch.py:247]: this is a kernel feature, so it usually common for all linux distros.

**Comment by liangan1** [comment]
@jgong5 ok

File: intel_pytorch_extension_py/launch.py (hunk 1 above)
Comment (liangan1) [reply intel_pytorch_extension_py/launch.py:247]: ok. thx

**Pull request closed without merging**